app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -334,13 +331,12 @@ class ProfileEditActivityTest {
         3
       )
     ).use {
+      testCoroutineDispatchers.runCurrent()
       onView(withId(R.id.profile_edit_allow_download_switch)).check(matches(isChecked()))
     }
   }
 
   @Test
-  // TODO(#973): Fix ProfileEditActivityTest
-  @Ignore
   fun testProfileEditActivity_configurationChange_startActivityWithUserHasDownloadAccess_checkSwitchIsChecked() { // ktlint-disable max-line-length
```

Comment:
We should remove `ktlint` comment by making the naming within the character limit.

Suggestion:
`fun testProfileEdit_configurationChange_startWithUserHasDownloadAccess_checkSwitchIsChecked() {`

Reply:
DONE